spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
What can be better instead would be adding `directive` boolean flag in this extension of `ExpressionStatement`. That would make it easy to check that statement is a directive (which is the entire point of such kind of addition) without breaking backward compatibility and at the same time data won't be duplicated and thus no problems with synchronization on node changes.